addons/sevenrings/CfgVehicles_I_Eastern.hpp

```diff
@@ -0,0 +1,776 @@
+        MACRO_EAST_DEFAULT_WEAPON
+    };
+    respawnWeapons[] = {
+        "TACU_SevenRings_W_East_Scorpion_Evo",
+        MACRO_EAST_DEFAULT_WEAPON
+    };
+    magazines[] = {
+        mag_8("CUP_30Rnd_9x19_EVO"),
+        MACRO_EAST_DEFAULT_MAGAZINE
+    };
+    respawnMagazines[] = {
+        mag_8("CUP_30Rnd_9x19_EVO"),
+        MACRO_EAST_DEFAULT_MAGAZINE
+    };
+    editorPreview = QPATHTOF(ui\SevenRings_U_I_Eastern_CQBSpecialist_03.jpg);
+};
+
+class TACU_SevenRings_U_I_Eastern_Marksman_01: TACU_SevenRings_U_I_Eastern_Rifleman_01 {
+    displayName = "Marksman (SVD)";
+    role = "Marksman";
+    sensitivity = 2.25;
+    uniformClass = "CUP_O_TKI_Khet_Partug_01";
+    linkedItems[] = {
+        DEFAULT_ITEMS_RADIO,
+        "milgp_v_marciras_marksman_cb"
+    };
+    respawnLinkedItems[] = {
+        DEFAULT_ITEMS_RADIO,
+        "milgp_v_marciras_marksman_cb"
+    };
+    weapons[] = {
+        "TACU_SevenRings_W_East_SVD",
+        MACRO_EAST_DEFAULT_WEAPON
+    };
+    respawnWeapons[] = {
+        "TACU_SevenRings_W_East_SVD",
+        MACRO_EAST_DEFAULT_WEAPON
+    };
+    magazines[] = {
+        mag_8("tacgt_10Rnd_762x54_SVD_AP_Mag"),
+        MACRO_EAST_DEFAULT_MAGAZINE
+    };
+    respawnMagazines[] = {
+        mag_8("tacgt_10Rnd_762x54_SVD_AP_Mag"),
+        MACRO_EAST_DEFAULT_MAGAZINE
+    };
+    editorPreview = QPATHTOF(ui\SevenRings_U_I_Eastern_Marksman_01.jpg);
+};
+
+class TACU_SevenRings_U_I_Eastern_Marksman_02: TACU_SevenRings_U_I_Eastern_Marksman_01 {
+    displayName = "Marksman (Cyrus)";
+    uniformClass = "CUP_U_C_racketeer_01";
+    weapons[] = {
+        "TACU_SevenRings_W_East_Cyrus",
+        MACRO_EAST_DEFAULT_WEAPON
+    };
+    respawnWeapons[] = {
+        "TACU_SevenRings_W_East_Cyrus",
+        MACRO_EAST_DEFAULT_WEAPON
+    };
+    magazines[] = {
+        mag_8("10Rnd_93x64_DMR_05_Mag"),
+        MACRO_EAST_DEFAULT_MAGAZINE
+    };
+    respawnMagazines[] = {
+        mag_8("10Rnd_93x64_DMR_05_Mag"),
+        MACRO_EAST_DEFAULT_MAGAZINE
+    };
+    editorPreview = QPATHTOF(ui\SevenRings_U_I_Eastern_Marksman_02.jpg);
+};
+
+class TACU_SevenRings_U_I_Eastern_Marksman_03: TACU_SevenRings_U_I_Eastern_Marksman_01 {
+    displayName = "Marksman (FN FAL)";
+    uniformClass = "CUP_O_TKI_Khet_Partug_04";
+    weapons[] = {
+        "TACU_SevenRings_W_East_FNFAL_5061",
+        MACRO_EAST_DEFAULT_WEAPON
+    };
+    respawnWeapons[] = {
+        "TACU_SevenRings_W_East_FNFAL_5061",
+        MACRO_EAST_DEFAULT_WEAPON
+    };
+    magazines[] = {
+        mag_8("tacgt_20Rnd_762x51_FAL_AP_Mag"),
+        MACRO_EAST_DEFAULT_MAGAZINE
+    };
+    respawnMagazines[] = {
+        mag_8("tacgt_20Rnd_762x51_FAL_AP_Mag"),
+        MACRO_EAST_DEFAULT_MAGAZINE
+    };
+    editorPreview = QPATHTOF(ui\SevenRings_U_I_Eastern_Marksman_03.jpg);
+};
+
+class TACU_SevenRings_U_I_Eastern_AA_01: TACU_SevenRings_U_I_Eastern_Rifleman_01 {
+    displayName = "Rifleman AA (Stinger)";
```

Comment:
Would personally think `Anti-Air Specialist (Stinger)` or something works better here, to keep inline with how BI names stuff as well (I don't think any other faction from BI or CUP or whatever has a Rifleman AA, but I may be wrong).
Can double check what BI uses for NATO or something and just mostly copy that if need be.